src/els_db.erl

```diff
@@ -36,6 +37,8 @@ install(NodeName, BaseDir) ->
 
 -spec ensure_db() -> ok.
 ensure_db() ->
+  %% Avoid mnesia overload while indexing
+  application:set_env(mnesia, dump_log_write_threshold, 50000),
```

Comment:
I added this to avoid all the noise in the logs.

Reply:
Should we move this line to the `install/2` function where we also specify the Mnesia `dir`? It feels more natural to do it there than in the function responsible to create the schema.

Reply:
Moved 😄